Management/src/ActuatorWeb/appsettings.Development.json

```diff
@@ -0,0 +1,41 @@
+{
+  // Steeltoe: Add schema to get auto-completion.
+  "$schema": "https://steeltoe.io/schema/latest/schema.json",
+  "DetailedErrors": true,
```

Comment:
This isn't in the default template, why do we need this?

Reply:
This is part of the razor pages template

Reply:
Right, I just overlooked it.